src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -598,6 +598,9 @@
   <data name="Unknown" xml:space="preserve">
     <value>Unknown</value>
   </data>
+  <data name="Extract_Local_Method" xml:space="preserve">
+    <value>Extract Local Method</value>
+  </data>
   <data name="Extract_Method" xml:space="preserve">
     <value>Extract Method</value>
```

Comment:
this should likely be "Extract method" as well.  Not sure why 'Method' is capitalized.  Before changing this though, talk to the team about which is preferred: "Title Cased", or "Sentence cased".  And made both consistent with each other :)